newIDE/app/src/ResourcesList/BrowserResourceExternalEditors.js

```diff
@@ -14,41 +367,74 @@ const editors: Array<ResourceExternalEditor> = [
     createDisplayName: t`Create with Piskel`,
     editDisplayName: t`Edit with Piskel`,
     kind: 'image',
-    edit: async () => {
-      sendExternalEditorOpened('piskel');
-      Window.showMessageBox(
-        'This feature is only supported in the desktop version for now!\nDownload it from GDevelop website.'
-      );
+    edit: async options => {
+      if (options.getStorageProvider().internalName !== 'Cloud') {
+        const { i18n } = options;
+        showWarningBox(i18n._(cloudProjectWarning), {
+          delayToNextTick: true,
+        });
+        return null;
+      }
 
-      return null;
+      const externalEditorWindow = immediatelyOpenLoadingWindowForExternalEditor();
+      return await editWithBrowserExternalEditor({
+        options,
+        externalEditorWindow,
+        externalEditorName: 'piskel',
+        defaultName: 'New image',
```

Comment:
Here we talk about an image (to prove my point about pixel ≠ image)

Reply:
Applied the change 👍